reports/parser.go

```diff
@@ -34,5 +32,5 @@ func parseCommandOuput(commandOutput string, packageManager string) []Package {
 		return parseLinuxOutput(commandOutput)
 	}
 
-	return []Package{}
+	return nil
```

Comment:
Can we return an empty map instead so we still have a valid json even we fail to get the list of packages?